interface/globals.php

```diff
@@ -192,6 +192,15 @@ function undoMagicQuotes($array, $topLevel=true) {
 // Include the version file
 require_once (dirname(__FILE__) . "/../version.php");
 
+// The logging level for common/logging/logger.php
+$GLOBALS["log_level"] = "DEBUG"; // (value can be TRACE, DEBUG, INFO, WARN, or ERROR)
+
+// Should Doctrine run in "dev mode"?
+$GLOBALS["doctrine_dev_mode"] = false;
```

Comment:
My rationale was it's best to keep configuration external, but in this case it's really an implementation detail. I will move it into the class.